sdk/eventhub/event-hubs/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 5.7.0-beta.1 (Unreleased)
+## 5.7.0-beta.1 (2021-11-12)
```

Comment:
empty ones can be removed
![image](https://user-images.githubusercontent.com/10452642/141598348-202d1cb9-f44c-4c06-8cf8-e494cc06b440.png)

Reply:
Man you saved me a lot of time would have been lost running the release pipeline twice! Thanks!